hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.test.tsx

```diff
@@ -34,16 +35,12 @@ describe("<Navbar />", () => {
         const handleLogoutSpy = jest.fn();
 
         const { getByText } = render(
-            <UnconnectedNavbar
-                cartQuantity={cartQuantity}
-                pathname={"/"}
-                logout={handleLogoutSpy}
-            />
+            <UnconnectedNavbar pathname={"/"} logout={handleLogoutSpy} />
         );
 
         // click logout button
         const logOutButton = getByText("Logout").closest("button");
-        fireEvent.click(logOutButton);
+        fireEvent.click(logOutButton!);
```

Comment:
remove ! lol

Reply:
I got this error, `Argument of type 'HTMLButtonElement | null' is not assignable to parameter of type 'Document | Node | Element | Window'.`, so when I use the ! mark it goes away, lol

Reply:
Ah I see, this test is weird it doesn't need the .closest() on line 42, you can remove that (and remove the exclamation mark too)